sprocs/workspaces_state_update.sql

```diff
@@ -21,16 +24,8 @@ BEGIN
         w.version INTO workspace_version;
 
     INSERT INTO workspace_logs
-        (date, level, message, workspace_id, workspace_version)
+        (date, level, message, workspace_id, version)
     VALUES
-        (now(),
-         'info'::enum_log_level,
-         CASE
-             WHEN (workspace_message = '') IS NOT FALSE -- either empty or null
-             THEN 'state:' || workspace_state
-             ELSE 'state:' || workspace_state || ', message:' || workspace_message
-         END,
-         workspace_id,
-         workspace_version);
+        (now(), workspace_level::enum_log_level, workspace_message, workspace_id, workspace_version);
```

Comment:
I think we shouldn't need `::enum_log_level` here if the type is correct in the function?